### DISCUSSION TITLE
поле тип "пользователь" ???

### DISCUSSION THREAD

**Discussion by nikidjan** [totumonline/totum-ru-issues-and-discussions#477] [Как это сделать?] [answered]
не смог найти информации о том как работать с пользовательской таблицей, как например у таблицы сделать что то на подобии:
поле тип "юзер" в котором выбираем существующих юзеров (но тут в принципе вроде понятно что селект из таблицы с юзерами)
А, САМОЕ ГАЛВНОЕ: далее на основе этого настроить доступ пользователю к определенным записям в таблице (например таб "задачи"  где он выбран "исполнителем") или же например доступ к конкретным полям

**Comment by erpic-ru** [comment, marked as answer]
Читайте про префильтры
https://ru.docs.totum.online/prefilter
https://ru.docs.totum.online/how-does-access-to-change-and-prefilter-visibility-affect-filtering
https://www.youtube.com/watch?v=cPcoB_3jl8o

**Reply by nikidjan** [reply]
ааа, точно! )) благодарю!!
